src/variety/cbblock.js

```diff
@@ -645,15 +645,15 @@
 			"A block contains a single color."
 		],
 		bkSubGt2: [
-			"同じ色のマスの塊が3個以上入っているブロックがあります。",
+			"同じ色のマスのかたまりが3個以上入っているブロックがあります。",
```

Comment:
This modification means exactly the same thing, only the letters have been changed. ("塊"="かたまり")

but, in many rule descriptions, the notation "カタマリ" is used instead of "かたまり" or "塊".
So,  **"同じ色のマスのカタマリが3個以上入っているブロックがあります。" is preferred.**